phpBB/config/default/container/services_twig.yml

```diff
@@ -54,6 +54,13 @@ services:
         tags:
             - { name: twig.extension }
 
+    template.twig.extensions.icon:
```

Comment:
Why you don't add this services in another file better?
This file is included in the installer environment, and probably can cause problems with some dependencies

Reply:
The dependency is `user`, that should be available, right?

But I know some other icon extensions have the similar problem, so perhaps it's a good idea to start using a separate config file, something like `services_twig_extensions.yml` so we can only include those in the production environment?